src/Resources/contao/forms/FormCaptcha.php

```diff
@@ -147,27 +147,44 @@ public function validate()
 
 
 	/**
-	 * Generate the captcha question
-	 *
-	 * @return string The question string
+	 * Generate the captcha values and store them in the session
 	 */
-	protected function getQuestion()
+	protected function generateCaptcha()
 	{
+		if ($this->arrCaptcha !== null) {
```

Comment:
This could be omitted if we added the more explicit check I have suggested below.